app/assets/javascripts/tests/_mobile_sessions.test.js

```diff
@@ -165,6 +165,32 @@ test('selectSession after successfully fetching calls drawSession.drawMobileSess
   t.end();
 });
 
+test('selectSession after successfully fetching undraws all sessions', t => {
+  const drawSession = mock('clear');
+  const sessions = []
+  const mobileSessionsService = _mobileSessions({ drawSession, sensors: { sensors: { 123: { sensor_name: 'sensor_name' } } } });
+  mobileSessionsService.sessions = sessions
+
+  mobileSessionsService.selectSession(1);
+
+  t.true(drawSession.wasCalledWith(sessions));
+
+  t.end();
```

Comment:
It's a bit noisy this tests. Could we remove some more and just leave the essentials?